src/python/pants/backend/python/packaging/pyoxidizer/target_types.py

```diff
@@ -40,16 +41,23 @@ class PyOxidizerOutputPathField(OutputPathField):
     )
 
 
-class PyOxidizerBinaryNameField(OutputPathField):
+class PyOxidizerBinaryNameField(StringField, AsyncFieldMixin):
```

Comment:
This field should not have been subclassing `OutputPathField` since the field `PyOxidizerOutputPathField` is already a subclass of `OutputPathField`.